okio-resourcefilesystem/build.gradle

```diff
@@ -0,0 +1,33 @@
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
```

Comment:
Probably can be .jvm, right?

Reply:
Flipping it around, how could I make this for non JVM usage? in builds or in binaries?

Also, any opinions of the utility of this PR? Am I over engineering?